src/main/java/services/notificationsending/EmailNotificationPresenter.java

```diff
@@ -30,6 +30,11 @@ public EmailNotificationPresenter(boolean enabled, String senderEmail,
         this.subject = subject;
     }
 
+    public EmailNotificationPresenter() {
+        this(true, "TimeTeam207@gmail.com",
+                "CSC207CSC207!", "lin507892@gmail.com", "CSC207 Project Time Notification");
```

Comment:
Does your email have to be hard coded into the default constructor? could it just be null instead?

Reply:
Oh that's in emailnotification PR where I added to test it works.
This should be deleted now.